### ISSUE TITLE
Instructions

### ISSUE THREAD

**Issue by rohinrohin** [QinganZhao/Deep-Learning-Based-Structural-Damage-Detection#4]
Hi QinganZhao!

Thanks for sharing this project. I am working on a similar project who's application depends on this project. It would be great if you could give me some instruction to just run the program to detect cracks, preferably using the fcnn. Or if you could tell me about the structure of your project. 

Any help is appreciated!

Thanks :)

**Comment by QinganZhao** [comment]
Hi,

To detect cracks, you may either use CNN classifier with a sliding window method or ROI method (like fcnn). I would recommend use fcnn. To begin with, label your training set manually (ie, circle the crack area of each sample. Or you could use a small set of samples to "pre" train a model and generate the rest). Then use any of the tools that could implement fcnn. I used Caffe in this project (a little old maybe, but it is simple and the structure  is clear).

The Python version (pycaffe) can be found at: https://github.com/rbgirshick/py-faster-rcnn
The MATLAB version (matcaffe) can be found at: https://github.com/ShaoqingRen/faster_rcnn

You shall be able to train your data by just replacing the data in their demo. You can also fine-tune the CNN which can be found clearly in their code.
